templates/model.js

```diff
@@ -1,6 +1,11 @@
 const mongoose = require('mongoose');
 const Schema   = mongoose.Schema;
 
-const {schemaName} = new Schema({fields});
+const {schemaName} = new Schema(
+        {fields},
+        {
+            timestamps: true,
+        }
```

Comment:
This is a good idea, but it must be an option that can be activated or not.

Reply:
removed timestamp, it will be another PR